style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+h3 {
+  font-size: 18px;
+  font-weight: 700;
+  margin: 0;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin: 46px;
+}
+
+p {
+  margin: 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+img {
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 14px 24px;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+}
+
+.menu {
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+  background-image: url('images/mobile_bg.png');
```

Comment:
Kindly crosscheck that the url of the background image is correct. Currently, there seems not to be a `mobile_bg.png` file in the images folder.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198837242-346ffca7-bb93-40ee-a3ff-dc1a25d497c1.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/198837284-5e9d44ef-46c0-40e0-ab7e-72293cc615be.png)